### PR TITLE
Add support for tagged domains in list creation script

### PR DESCRIPTION
**Pull request by englehardt**
This PR adds support for parsing our the `fingerprinting` and `cryptomining` portions of the Disconnect list. I checked the logs for the generated lists and spot checked them against the domains generated by [a separate parser](https://github.com/mozilla/trackingprotection-tools/blob/master/DisconnectParser.py).

### CODE REVIEW

**Comment by englehardt** [comment]
Fixes #59

File: lists2safebrowsing.py [hunk 1]
```diff
@@ -182,6 +205,13 @@ def find_hosts(blocklist_json, allow_list, chunk, output_file, log_file,
     chunk: The chunk number to use.
     output_file: A file-handle to the output file.
     log_file: A filehandle to the log file.
+    which_dnt: A filter to restrict output to section of the list with the
+        specified DNT tag.
+    list_categories : A filter to restrict output to the specified top-level
+        categories.
+    name : The section name from `shavar_list_creation.ini`
+    which_tag : A filter to restrict output to sections of the list with the
```
[the remark this answers is not in the document]
Comment (englehardt) [reply lists2safebrowsing.py]: Note that if we decide to support new tags that aren't on the current list we'll still need to add them to the `ALL_TAGS` constant at the top of the script. The way the script is written right now we need to know the set of possible tags so we can try to pop them.

It's possible to work around the need to know the tags prior to parsing the list, but it requires a bit of refactoring in the way we parse the list. I did that in [my parser](https://github.com/mozilla/trackingprotection-tools/blob/1547cc88d83715f7fb1f12a84116188941ad556d/DisconnectParser.py#L125-L130) and ideally we'd move to that than pay the refactoring cost here.

[the remark this answers is not in the document]
Comment (englehardt) [reply lists2safebrowsing.py]: I also updated https://github.com/mozilla-services/shavar-list-creation-config/pull/35 to follow this new format.

[the remark this answers is not in the document]
Comment (groovecoder) [reply lists2safebrowsing.py]: 👍 to skip the full refactor in favor of switching to your parser later.

**Pull request merged**